tests/models/whisper/test_modeling_whisper.py

```diff
@@ -745,6 +755,159 @@ def _create_and_check_torchscript(self, config, inputs_dict):
 
             self.assertTrue(models_equal)
 
+    @is_pt_flax_cross_test
+    def test_equivalence_pt_to_flax(self):
```

Comment:
Don't think there's a need to repeat this PT-FX equivalence test since we already include it in `test_modeling_flax_whisper`, feel free to remove the Flax tests from the PT modelling test file